pkg/v21/key/error_internal.go

```diff
@@ -0,0 +1,21 @@
+package key
+
+import (
+	"strings"
+
+	"github.com/giantswarm/microerror"
+)
+
+const (
+	errorSentence = "the server could not find the requested resource (get chartconfigs.core.giantswarm.io)"
```

Comment:
```suggestion
	errorText = "the server could not find the requested resource (get chartconfigs.core.giantswarm.io)"
```

🇺🇸 nit but if there is no `.` it's not a sentence. 😆

Reply:
omg, that's so detailed! Thanks.